packages/vue/__tests__/Transition.spec.ts

```diff
@@ -1296,23 +1330,28 @@ describe('e2e: Transition', () => {
 
         await nextFrame()
         expect(await html('#container')).toBe(
-          '<div class="test v-enter-active v-enter-to">one</div>'
+          '<div class="test v-enter-active v-enter-to" style="">one</div>'
         )
         await transitionFinish()
-        expect(await html('#container')).toBe('<div class="test">one</div>')
+        expect(await html('#container')).toBe(
+          '<div class="test" style="">one</div>'
+        )
 
         // leave
         await classWhenTransitionStart()
         await nextFrame()
         expect(await html('#container')).toBe(
-          '<div class="test v-leave-active v-leave-to">one</div>'
+          '<div class="test v-leave-active v-leave-to" style="">one</div>'
         )
         await transitionFinish()
+        await nextFrame()
```

Comment:
**Why add `await nextFrame()` ?**

```typescript
  const buffer = 5
  const transitionFinish = (time = duration) => timeout(time + buffer)
```
Because the 5 milliseconds can not ensure the next frame refresh.

**Why there is no problem before ?**
Because the leave transition will be broken when effect on Suspense component. Tell me if need a reproduction link.